lib/Sema/CodeSynthesis.cpp

```diff
@@ -1332,6 +1685,10 @@ void swift::maybeAddAccessorsToVariable(VarDecl *var, TypeChecker &TC) {
     return;
   }
 
+  // Local variables don't otherwise get accessors.
+  if (var->getDeclContext()->isLocalContext())
```

Comment:
I test some local vars with behaviors, yeah. We ban `lazy` in local contexts so it doesn't yet matter in trunk.